### PR TITLE
[15.0][MIG] sale_timesheet_invoice_description: Migration to version 15.0

### PR DESCRIPTION
**Pull request by LudLaf**
@Tecnativa TT38402

### CODE REVIEW

**Comment by LudLaf** [comment]
@carlosdauden
@chienandalu 
@pedrobaeza 
Please review.

File: sale_timesheet_invoice_description/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
 To configure this module, you need to:
 
-#. Go to *Timesheets -> Configuration -> Settings -> Billing* and set
-   Default Timesheet Invoice Description.
+* Go to **Timesheets > Configuration > Settings** and select an option from
```
Comment (pedrobaeza) [inline sale_timesheet_invoice_description/readme/CONFIGURE.rst]: Don't change the numbered list format.

File: sale_timesheet_invoice_description/tests/test_sale_timesheet_description.py [hunk 2]
```diff
@@ -8,12 +8,12 @@
...
 
 
-class TestSaleTimesheetDescription(common.SavepointCase):
+class TestSaleTimesheetDescription(common.TransactionCase):
     @classmethod
     def setUpClass(cls):
         super(TestSaleTimesheetDescription, cls).setUpClass()
         # Make sure user is in English
-        cls.env["res.lang"].load_lang("en_US")
+        cls.env["res.lang"]._activate_lang("en_US")
```
Comment (pedrobaeza) [inline sale_timesheet_invoice_description/tests/test_sale_timesheet_description.py]: There's no need of loading this language, as it's already pre-loaded.

**Review by pedrobaeza — APPROVED** [review]

File: sale_timesheet_invoice_description/tests/test_sale_timesheet_description.py (hunk 2 above)
Comment (LudLaf) [reply sale_timesheet_invoice_description/tests/test_sale_timesheet_description.py]: Fixed

File: sale_timesheet_invoice_description/readme/CONFIGURE.rst (hunk 1 above)
Comment (LudLaf) [reply sale_timesheet_invoice_description/readme/CONFIGURE.rst]: Fixed

**Review by pedrobaeza — APPROVED** [review]

**Review by jabelchi — APPROVED** [review]
Functional testing OK. 
(Runboat doesn't start. Tested on premise, using a virtual machine)

**Pull request merged**